src/server/controllers/Ride.js

```diff
@@ -31,18 +31,19 @@ export const addRide = function (req, res) {
       // carvisUserId -- to query the user table for tokens etc.
       let userId = ride.userId;
 
+      // TODO: Redis retrieve user -- if not found, do DB query.
       let dbURL = 'http://' + CARVIS_API + '/users/' + userId;
       console.log('pre db get', vendor, userId, dbURL, rideId);
 
-      return getUserAndRequestRide(dbURL, origin, destination, partySize, rideId, vendor)
+      return getUserAndRequestRideDB(dbURL, origin, destination, partySize, rideId, vendor);
     })
     .catch((err) => res.status(400)
       .json(err)); // add catch for errors.
 };
 
-const getUserAndRequestRide = (dbURL, origin, destination, partySize, rideId, vendor) => {
+const getUserAndRequestRideDB = (dbURL, origin, destination, partySize, rideId, vendor) => {
 
-  fetch(dbURL, {
+  return fetch(dbURL, {
```

Comment:
@cpruijsen why is there a return in the fetch call? i ask cuz i might have to do something similar elsewhere, but i wanna make sure i understand it correctly